checkstyle.xml

```diff
@@ -41,7 +41,7 @@
 			<property name="ignoreComments" value="true"/>
 		</module>
 		<module name="RegexpSinglelineJava">
-			<property name="format" value="[^\s]\s+$"/>
+			<property name="format" value=".*\s+$"/>
```

Comment:
Is this regex run against the whole line, or does it match substrings? eg. does `\s+$` work to match whitespace at EOL?

Reply:
This matches substrings. I expanded the original regex to work on both lines with content `[^\s]\s+$` and without content `.*\s+$`.

However, this rule would work if we wanted two separate rules for this instead:
```xml
<module name="RegexpSinglelineJava">
	<property name="format" value="^\s+$"/>
	<property name="message" value="Empty lines should not contain whitespace"/>
	<property name="ignoreComments" value="true"/>
</module>
```